01_fib/saurabhvyas9/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// Abs returns the absolute value of x.
+func Abs(x int) int {
+	if x < 0 {
+		return -x
+	}
+	return x
+}
+
+// Returns Fibonacci and Negafibonacci series
+func fib(n int) {
+	m := Abs(n)
```

Comment:
This works - feel free to leave it as is. You could get rid of `abs` and simplify it to:
```go
	sign := // 1 for positive n ; -1 for negative n
	n1, n2 := 0, 1
	for j := 0; j < n*sign; j++ {
		n1, n2 = n2, n1+sign*n2
		// print n1
	}
```

Reply:
Removing the <Abs> would require me to put the if condition while declaring sign variable. If condition is going to be used in both ways. 
No change done.